### DISCUSSION TITLE
Azure Serverless howto/documentation request

### DISCUSSION THREAD

**Discussion by semopz** [blitz-js/blitz#1215] [Q&A]
I'd like to ask how practical would it be to host blitz apps on Azure and are any guides planned for such deployments? The first challenge I could think of is that Cosmos DB (Azure's serverless db service) does not support MySQL or PostgreSQL.

## WHY????
Sure, I know Azure isn't everyone's first choice so here are a few reasons
* Inertia - company I'm in already uses Azure so billing, support and other boring bits are already taken care of
* UK datacentres. I recently worked for a company that refused to use a service I proposed because its parent company wasn't based in the UK (despite the app and data being hosted on Azure).
* Most things I'll need for my app are in one place (auth, emails, CI/CD) even if better standalone products/services exist elsewhere. Again, inertia in the environment I'm in

**Comment by flybayer** [comment]
Yeah you can totally host on Azure! If you want to use Cosmos, you won't be able to use prisma, but that's no problem for Blitz.

If you get it figured out, it'd be awesome if you can make a PR to the docs repo to add an azure deployment guide.
